src/dbSta/src/dbSta.tcl

```diff
@@ -79,16 +79,28 @@ proc highlight_path { args } {
   }
 }
 
-define_cmd_args "report_cell_usage" {[-verbose]}
+define_cmd_args "report_cell_usage" {[-verbose] [module_inst]}
```

Comment:
Doc update?

Reply:
it's a strange location for the docs, but updated.